src/site/content/en/blog/bluetooth/index.md

```diff
@@ -558,7 +556,6 @@ Hero image by [SparkFun Electronics from Boulder, USA].
 [Promises tutorial]: /promises
 [Arrow functions]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Functions/Arrow_functions
 [Bluetooth GATT Battery Service]: https://www.bluetooth.com/specifications/gatt/
-[the standardized Bluetooth GATT services]: https://www.bluetooth.com/specifications/assigned-numbers/
 [standardized Battery Level Characteristic]: https://www.bluetooth.com/specifications/gatt/
```

Comment:
This link is state and redirects to https://www.bluetooth.com/specifications/specs/, where you then need to search for the specification you are interested in. It may be worth adding some text explaining how to find these specifications since they can't be linked directly.